channels/vn.m3u

```diff
@@ -29,3 +29,75 @@ http://thtg.vn:8001/thtg.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vati Lao TV
+https://livefta.malimarcdn.com/ftaedge00/vatilaotv.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Houng Fa
+https://livefta.malimarcdn.com/ftaedge00/houngfa.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Dhammasapha TV
+https://livefta.malimarcdn.com/ftaedge00/dhammasapha.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Khomsanh TV
+https://livefta.malimarcdn.com/ftaedge00/lstv2.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao Heritage Foundation TV
+https://livefta.malimarcdn.com/ftaedge00/laoheritagetv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Supreme Master TV
+https://lbs-us1.suprememastertv.com/720p.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",PNTV
+https://livefta.malimarcdn.com/ftaedge00/pntv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lone Sinh Lao TV
+https://livefta.malimarcdn.com/ftaedge00/lonesinhlao.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Laos Planet TV
+https://livefta.malimarcdn.com/ftaedge00/laosplanet.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao Net TV
+https://livefta.malimarcdn.com/ftaedge00/laonet.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",LS TV
+https://livefta.malimarcdn.com/ftaedge00/laostv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Look Thoong TV
+https://livefta.malimarcdn.com/ftaedge00/lookthoongtv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Uniquely Thai
+https://livefta.malimarcdn.com/ftaedge00/uniquely.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vanphenh TV
+https://livefta.malimarcdn.com/ftaedge00/vanphenhtv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Sieng Khaen Lao TV
+https://livefta.malimarcdn.com/ftaedge00/vlctemple.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao Champa TV 1
+https://livefta.malimarcdn.com/ftaedge00/laochampa1.sdp/playlist.m3u8
```

Comment:
Language of this channel is not only Vietnamese, I see some English parts.
![vlcsnap-2021-05-31-17h33m34s132](https://user-images.githubusercontent.com/30985701/120215706-554c0500-c236-11eb-87eb-aa218c0661e7.png)